protocol/transport/filter.go

```diff
@@ -15,8 +15,10 @@ type Filter struct {
 	// Identity is the public key of the other recipient for non-public filters.
 	// It's encoded using encoding/hex.
 	Identity string `json:"identity"`
-	// Topic is the whisper topic
-	Topic types.TopicType `json:"topic"`
+	// PubsubTopic is the waku2 pubsub topic
+	PubsubTopic string `json:"pubsubTopic"`
+	// ContentTopic is the waku topic
+	ContentTopic types.TopicType `json:"topic"`
```

Comment:
What is the chance of a community becoming partitioned / separated if users use an older client that uses any version of `status-go` before this change is introduced?

Reply:
> not making the sharding functionality available for at least 2 versions to give chance for users to migrate to newer status-go, or introducing a deprecation notice in the clients to force them to upgrade it?

That is probably the most realistic approach, it is what we do for other major breaking changes.